src/python_testing/TC_ECOINFO_2_2.py

```diff
@@ -15,32 +15,97 @@
 #    limitations under the License.
 #
 
+# See https://github.com/project-chip/connectedhomeip/blob/master/docs/testing/python.md#defining-the-ci-test-arguments
+# for details about the block below.
+#
+# === BEGIN CI TEST ARGUMENTS ===
+# test-runner-runs:
+#   run1:
+#     app: examples/fabric-admin/scripts/fabric-sync-app.py
+#     app-args: --app-admin=${FABRIC_ADMIN_APP} --app-bridge=${FABRIC_BRIDGE_APP} --stdin-pipe=dut-fsa-stdin --discriminator=1234
+#     script-args: >
+#       --PICS src/app/tests/suites/certification/ci-pics-values
+#       --storage-path admin_storage.json
+#       --commissioning-method on-network
+#       --discriminator 1234
+#       --passcode 20202021
+#       --string-arg th_server_app_path:${ALL_CLUSTERS_APP} dut_fsa_stdin_pipe:dut-fsa-stdin
+#       --trace-to json:${TRACE_TEST_JSON}.json
+#       --trace-to perfetto:${TRACE_TEST_PERFETTO}.perfetto
+#     script-start-delay: 5
+#     factoryreset: true
+#     quiet: false
+# === END CI TEST ARGUMENTS ===
+
+import asyncio
+import logging
+import random
+import tempfile
+
 import chip.clusters as Clusters
 from chip.interaction_model import Status
 from matter_testing_support import MatterBaseTest, TestStep, async_test_body, default_matter_test_main
 from mobly import asserts
+from TC_MCORE_FS_1_1 import AppServer
 
 _DEVICE_TYPE_AGGREGGATOR = 0x000E
 
 
 class TC_ECOINFO_2_2(MatterBaseTest):
 
+    def setup_class(self):
+        super().setup_class()
+
+        self.th_server = None
+        self.storage = None
+
+        # Get the path to the TH_SERVER app from the user params.
+        if th_server_app := self.user_params.get("th_server_app_path"):
+            # Create a temporary storage directory for keeping KVS files.
+            self.storage = tempfile.TemporaryDirectory(prefix=self.__class__.__name__)
+            logging.info("Temporary storage directory: %s", self.storage.name)
+
+            # Get the named pipe path for the DUT_FSA app input from the user params.
+            if dut_fsa_stdin_pipe := self.user_params.get("dut_fsa_stdin_pipe"):
```

Comment:
If this is CI we should be making sure that a value is provided here for `dut_fsa_stdin_pipe`

Reply:
Fixed